src/dotnet/APIView/APIViewWeb/Startup.cs

```diff
@@ -99,7 +102,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<LanguageService, SwiftLanguageService>();
             services.AddSingleton<LanguageService, XmlLanguageService>();
 
-            services.AddAuthentication(options =>
+            if (Environment.IsDevelopment()) 
```

Comment:
I have another comment about this above. My only worry is  we are leaving a backdoor if we happen to enable dev mode in any of the deployed test env in the future. So adding an extra check will avoid that. One another option is to use key based authentication which can be passed from internal env and in local testing.